common/components/WalletDecrypt/Mnemonic.tsx

```diff
@@ -86,19 +88,21 @@ export default class MnemonicDecrypt extends Component<Props, State> {
   };
 
   public onMnemonicChange = (e: React.SyntheticEvent<HTMLTextAreaElement>) => {
```

Comment:
I know this isnt part of your PR, but it should be `React.FormEvent<HTMLTextAreaElement>`, then you won't need to typecast `e.target` as you can use `e.currentTarget.value` instead